src/amo/components/SearchFilters/index.js

```diff
@@ -114,15 +114,17 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
   doSearch(newFilters: SearchFiltersType) {
     const { clientApp, lang, history, pathname } = this.props;
 
-    if (newFilters.page) {
```

Comment:
Why is this change needed actually?

Reply:
It's not, anymore. Subsequent changes have rendered it unnecessary. Thanks for pointing it out.

Reply:
And... this code is back now 😉